src/components/top-header/top-header.tsx

```diff
@@ -13,7 +13,7 @@ export function TopHeader() {
           <Link href="/">
             <Group spacing="xs">
               <Image src="/logo.png" alt="logo" width={32} height={32} />
-              <Title size="2rem" align="left" color="red" fw={700}>
+              <Title size="2rem" align="left" color="red.8" fw={700}>
```

Comment:
For the Top Header component we should add the Admin button.

Reply:
Added Admin button.